cli/commands/edit/edit.go

```diff
@@ -0,0 +1,95 @@
+package edit
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+
+	"github.com/sensu/sensu-go/cli"
+	"github.com/sensu/sensu-go/cli/commands/create"
+	"github.com/sensu/sensu-go/cli/commands/helpers"
+	"github.com/spf13/cobra"
+)
+
+// vi is the default editor!
+const defaultEditor = "vi"
+
+func Command(cli *cli.SensuCli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "edit [RESOURCE NAME] [KEY1] ... [KEYN]",
+		Short: "Edit resources interactively",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) < 2 {
+				_ = cmd.Help()
+				return errors.New("invalid argument(s) received")
+			}
+			resourceType := args[0]
+			resourceKeyParams := args[1:]
+			format := cli.Config.Format()
+			switch format {
+			case "yaml", "wrapped-json":
+			default:
+				format = "yaml"
+			}
+			ctlArgs := []string{
+				resourceType,
+				"info",
+			}
+			ctlArgs = append(ctlArgs, resourceKeyParams...)
+			ctlArgs = append(ctlArgs, "--format")
+			ctlArgs = append(ctlArgs, format)
+			fmt.Println(ctlArgs)
+			b, err := exec.Command("sensuctl", ctlArgs...).CombinedOutput()
```

Comment:
I'd replace `"sensuctl"` with `os.Args[0]` so we don't assume the name of the executable nor its location (might not necessarily be in $PATH)